crates/web5/src/dids/methods/did_web/resolver.rs

```diff
@@ -43,10 +48,20 @@ impl Resolver {
     }
 
     pub async fn resolve(&self) -> Result<ResolutionResult, ResolutionMetadataError> {
-        let response = http_std::fetch(&self.http_url, None)
+        let headers: HashMap<String, String> = HashMap::from([
+            ("Host".to_string(), "{}".to_string()),
+            ("Connection".to_string(), "close".to_string()),
+            ("Accept".to_string(), "application/json".to_string()),
+        ]);
```

Comment:
do you think all of these headers are necessary? I've noticed you have added them throughout the various http calls. I'm guessing it's slightly more proper this way? just checking this isn'y copy/pasta from our roll-our-own http client code